etl/steps/data/meadow/ophi/2022-12-13/multidimensional_poverty_index.meta.yml

```diff
@@ -0,0 +1,104 @@
+dataset:
+  namespace: ophi
+  version: 2022-12-13
+  title: Multidimensional Poverty Index (MPI) 2022
+  publication_year: 2022
+  short_name: multidimensional_poverty_index
+  description: "The global Multidimensional Poverty Index (MPI) is an international\
```

Comment:
It could using multi-line strings too, e.g.

```
description: |
  The global Multidimensional Poverty Index (MPI) is an international
  measure of acute multidimensional poverty covering over 100 developing countries.
  ...
```

I guess this was created by walkthrough? Would you appreciate having it generated automatically by walkthrough or you don't mind?

Reply:
If this metadata is identical to the one from snapshot, I think it would be better if it's not duplicated here (unless you have made some manual changes). You could load the snapshot metadata directly.